qemu/tests/numa_stress.py

```diff
@@ -114,39 +113,37 @@ def run(test, params, env):
             os.remove(memory_file)
         utils_memory.drop_caches()
         if utils_memory.freememtotal() < tmpfs_size:
-            raise error.TestError("Don't have enough memory to execute this "
-                                  "test after %s round" % test_round)
-        error.context("Executing stress test round: %s" % test_round,
-                      logging.info)
+            test.error("Don't have enough memory to execute this "
+                       "test after %s round" % test_round)
+        error_context.context("Executing stress test round: %s" % test_round,
+                              logging.info)
         numa_node_malloc = most_used_node
         numa_dd_cmd = "numactl -m %s %s" % (numa_node_malloc, dd_cmd)
-        error.context("Try to allocate memory in node %s" % numa_node_malloc,
-                      logging.info)
+        error_context.context("Try to allocate memory in node %s" % numa_node_malloc,
+                              logging.info)
         try:
             utils_misc.mount("none", tmpfs_path, "tmpfs", perm=mount_fs_size)
             funcatexit.register(env, params.get("type"), utils_misc.umount,
-                                "none", tmpfs_path, "tmpfs")
+                                None, tmpfs_path, "tmpfs")
```

Comment:
The value should be `"none"` for keeping the same value as L125 because here we want to umount the specified target.

Reply:
Thanks for your review. I checked the code again, it turns out the root cause why the umount() doesn't work is in avocao-vt(utils_misc.py):
1150     if fstype not in ['nfs', 'smbfs', 'glusterfs', 'hugetlbfs']:
1151         if src:
1152             src = os.path.realpath(src)
1153         else:
1154             # Allow no passed src(None or "")
1155             src = ""
1156     mount_string = "%s %s .*%s %s" % (src, mount_point, fstype_mtab, perm)
I will close this pr and send a new one to avocado-vt.  Thanks!